src/Chainweb/Miner/Core.hs

```diff
@@ -113,15 +104,10 @@ mine _ h = BA.withByteArray initialTargetBytes $ \trgPtr -> do
             hashInternalFinalize ctxPtr (castPtr pow)
     {-# INLINE hash #-}
 
-    -- | `injectTime` and `injectNonce` make low-level assumptions about the
-    -- byte layout of a hashed `BlockHeader`. If that layout changes, these
-    -- functions need to be updated. The assumption allows us to iterate on new
-    -- nonces quickly.
+    -- | `injectNonce` makes low-level assumptions about the byte layout of a
```

Comment:
Maybe we can take this opportunity (or very soon hereafter) to address this ticket?

https://github.com/kadena-io/chainweb-node/issues/297

Reply:
We won't be doing this afterall.